scripts/create_new_beamline.py

```diff
@@ -0,0 +1,49 @@
+import asyncio
+
+from nsls2api.infrastructure import mongodb_setup
+from nsls2api.infrastructure.config import get_settings
+from nsls2api.models.beamlines import Beamline, ServiceAccounts
+from nsls2api.services import pass_service
+
+settings = get_settings()
+
+# CHANGE THIS TO THE BEAMLINE YOU WANT TO CREATE
+BEAMLINE_NAME = "CDI"
+
+
+async def main():
+    # Initialize Beanie
+    await mongodb_setup.init_connection(settings.mongodb_dsn)
+
+    pass_resources = await pass_service.get_pass_resources()
+    pass_ids = [r["ID"] for r in pass_resources if r["Code"] == BEAMLINE_NAME]
+    if len(pass_ids) > 1:
+        raise ValueError(f"Multiple pass IDs found for {BEAMLINE_NAME}: {pass_ids}")
+
+    # CHANGE THESE VALUES TO SUIT YOUR NEEDS
+    new_beamline = Beamline(
+        name=BEAMLINE_NAME,
+        long_name="Fancy New Beamline",
+        alternative_name="99-ID",
+        port="99-ID",
+        nsls2_redhat_satellite_location_name="Beamlines/99-ID ABC",
+        pass_id=str(pass_ids[0]),
+        pass_name="Beamline 99-ID",
+        network_locations="xf99id1",
+        github_org="NSLS2",
+        service_accounts=ServiceAccounts(
+            ioc="softioc-abc",
+            epics_services="epics-abc",
+            workflow="workflow-abc",
+            bluesky="bluesky-abc",
+            operator="xf99id",
+        ),
+    )
+    print(new_beamline)
+
+    # Uncomment this line to actually insert the new beamline into the database
+    # await new_beamline.insert()
+
+
+if __name__ == "__main__":
+    asyncio.run(main())
```

Comment:
Naive question.  Where do you run this function?  If over the network, what level of permissions does it require?

Reply:
Should we add a check to make sure that we don't accidentally overwrite a beamline by a simple typo (i.e., CSX vs CHX)?  Maybe having an optional argument to force overwriting?

This is a bigger PR for later.  just asking to make sure we are all on the same page.

Reply:
I also changed the example to TLA for the beamline - the line that actually updates the database and makes changes is actually commented out as to try and avoid accidental issues.  

This script was meant more as a helper starting point than a full solution.